string color
int32 frameResolutionHeight
int32 frameResolutionWidth
